cdk/jenkins/jenkins_leader.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, scope: core.Stack, id: str, cluster, vpc, worker, **kwargs) -
                 },
                 logging=ecs.LogDriver.aws_logs(
                     stream_prefix="Jenkinsleader",
-                    log_retention=logs.RetentionDays.ONE_WEEK
+                    retention=logs.RetentionDays.ONE_WEEK # BUG: defaulting to 'never'
```

Comment:
TESTING: I'm not sure if this actually fixes the problem. Still testing.
Symptom: in CloudWatch, it shows the 'expire' column as 'never' currently.